.gitignore

```diff
@@ -62,3 +66,6 @@ typings/
 # own stuff
 dist/
 dist-*/
+
+# Atlassian
```

Comment:
The `.idea` folder is for `PHPStorm`, not `Atlassian`. Right? Can we change this comment?

Reply:
You can open and edit this in WebStorm as well and other Atlassian products. I thought I use a more general term to cover this as well.

Reply:
Or maybe just a generic name for this block like "IDE files" or "local settings"?